### PR TITLE
Cross server evacuation messages now include the evacuation reason

### PR DESCRIPTION
**Pull request by JetBrody**
## About The Pull Request

Adds the emergency shuttle request reason to the cross-server message received upon shuttle arrival at Central Command.

_(Shoutout to Rohesie for your patience and advice on this, I'm mentioning you anyway.)_

## Why It's Good For The Game

The reason for calling the shuttle is mostly pointless at this point, at best used for an administrator to figure out why a shuttle was called, at worst used by the Captain to justify ending a long round to the crew. This will give the other stations some context behind why a neighboring station was evacuated (or at least that "shit is fucked time to leave.")

## Changelog
:cl: Hulkamania
add: Other stations will now receive your emergency shuttle request reason! Tell them just how much plasma you're breathing!
/:cl:

### CODE REVIEW

File: code/controllers/subsystem/ticker.dm [hunk 1]
```diff
@@ -62,6 +62,8 @@ SUBSYSTEM_DEF(ticker)
 	/// People who have been commended and will receive a heart
 	var/list/hearts
 
+	/// Why an emergency shuttle was called
+	var/emergency_reason
 /datum/controller/subsystem/ticker/Initialize(timeofday)
```
[the remark this answers is not in the document]
Comment (bobbahbrown) [reply code/controllers/subsystem/ticker.dm:67]: should ensure there's a line between that proc definition and your last var definition

File: code/controllers/subsystem/ticker.dm [hunk 2]
```diff
@@ -525,7 +528,10 @@ SUBSYSTEM_DEF(ticker)
 		if(STATION_DESTROYED_NUKE)
 			news_message = "We would like to reassure all employees that the reports of a Syndicate backed nuclear attack on [station_name()] are, in fact, a hoax. Have a secure day!"
 		if(STATION_EVACUATED)
-			news_message = "The crew of [station_name()] has been evacuated amid unconfirmed reports of enemy activity."
+			if(emergency_reason)
+				news_message = "[station_name()] has been evacuated after transmitting the following distress beacon:\n\n[emergency_reason]"
```
Comment (Jared-Fogle) [inline code/controllers/subsystem/ticker.dm:532]: Make sure you can't do XSS with this, like calling the shuttle with the reason `<h1>funny</h1>`

Comment (Jared-Fogle) [reply code/controllers/subsystem/ticker.dm:532]: On a look through of the code, it looks like it is html_encoded'd anyway, so nevermind.

Comment (JetBrody) [reply code/controllers/subsystem/ticker.dm:532]: Not an issue, it's handled through minor_announce which uses html_encode

Tested locally to be sure, just spits out the text
image

**Pull request merged**